apps/src/code-studio/progressRedux.js

```diff
@@ -14,7 +14,7 @@ const initialState = {
   currentLevelId: null,
   professionalLearningCourse: null,
   // a mapping of level id to result
-  progress: {},
+  levelProgress: {},
```

Comment:
It seemed really weird having state.progress.progress, and levelProgress seems like a more accurate description of this regardless.